Test_case/conn_test3.py

```
@@ -0,0 +1,156 @@
+    )
+    if not patch_resp.ok:
+        print(f"--- tusd PATCH failed ({patch_resp.status_code}) ---\n{patch_resp.text!r}")
+    patch_resp.raise_for_status()
+
+    print(f"[uploaded] {event['name']:<28} -> {location}")
+    return location
+
+
+def fetch_kafka_messages() -> str:
+    """Drain the topic from the beginning via the kafka-console-consumer CLI."""
+    result = subprocess.run(
```

Ruff (0.15.17)

[error] `subprocess` call: check for execution of untrusted input

(S603)